### PR TITLE
fix(source-linkedin-ads): revert migration to low code

### PR DESCRIPTION
**Pull request by darynaishchenko**
## What
oc: https://github.com/airbytehq/oncall/issues/6198
reverting low code migration due to outage in the low code source.

## Can this PR be safely reverted and rolled back?

- [ ] YES 💚
- [ ] NO ❌

### CODE REVIEW

**Review by girarda — APPROVED** [review]
:shipit:

File: docs/integrations/sources/linkedin-ads-migrations.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
 # LinkedIn Ads Migration Guide
 
+## Upgrading to 4.0.0
+
+Version 3.X.X introduced a regression in the connector that was reverted in 4.0.0. If you were using 3.X.X, please go through the migration steps. If you were still using 2.X.X, please upgrade to 4.0.0; after that, there is no additional action required.
```
Comment (katmarkham) [inline docs/integrations/sources/linkedin-ads-migrations.md:5]:
```suggestion
Version 3.X.X introduced a regression in the connector that was reverted in 4.0.0. If you were using 3.X.X, please go through the migration steps. If you were still using 2.X.X, please upgrade to 4.0.0; after that, there are no additional actions required.
```

**Review by katmarkham — APPROVED** [review]

**Comment by natikgadzhi** [comment]
@darynaishchenko @girarda do you have a plan on how to move forward with the migration? there are a few improvements that community folks would love to add, but waiting on the low-code migration first: 

- Leads stream: https://github.com/airbytehq/airbyte/pull/42101
- Organizations stream: https://github.com/airbytehq/airbyte/issues/44038

**Comment by darynaishchenko** [comment]
@lazebnyi Could you advise on @natikgadzhi's question?

**Comment by kev-datams** [comment]
thanks @natikgadzhi 
@lazebnyi , there is also the Sponsored Ads Content stream which is concerned: https://github.com/airbytehq/airbyte/pull/35655

**Comment by lazebnyi** [comment]
@natikgadzhi I'm currently working on fixing issues that were discovered after the release of the low-code version. ETA: Submit a PR with fixes and migration for review by Monday morning.

**Comment by kev-datams** [comment]
Hi @lazebnyi, do you have some new about your PR ? 🙏

**Comment by natikgadzhi** [comment]
We've only unblocked Serhii yesterday with regression tests, so it might take a little bit longer to clean things up.

**Comment by kev-datams** [comment]
Thanks for the update 🤞

**Pull request merged**